docs/apis-tools/zeebe-api-rest/sidebar-schema.js

```diff
@@ -0,0 +1,10 @@
+/** @type {import('@docusaurus/plugin-content-docs').SidebarsConfig} */
+
+module.exports = {
+  "Zeebe API (REST)": [
+    "apis-tools/zeebe-api-rest/zeebe-api-rest-overview",
+    {
+      Specifications: require("./specifications/sidebar.js"),
```

Comment:
I opted to put the generated docs inside a section named "Specifications". Among other reasons, this allows me to avoid dealing with a confusing conflict between the "Overview" doc, which we manage manually and will eventually contain more useful information for getting started with the API, and the generated "Introduction" doc, which is less useful but any manual changes to it would easily be overwritten during generation.